### PR TITLE
fix make_docs and qsub test

### LINKED ISSUE #1711: [DEFECT] type in qsub tests, and make_docs failed on sawtooth
### Thank you for the defect report

- [X] I am using the latest version of `RAVEN`.
- [X] I have read the [Wiki](https://github.com/idaholab/raven/wiki).
- [ ] I have created a [minimum, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example)
that demonstrates the defect.

### Defect Description

There is a type in qsub tests, and make_docs failed to build documents in sawtooth 

### Steps to Reproduce

check tests/cluster_tests/tests for test_mpi_fake 
sawtooth tests failed on devel for docs build 

### Expected Behavior

all qsub tests should pass, not skip 

### Screenshots and Input Files

_No response_

### OS

MacOS

### OS Version

_No response_

### Dependency Manager

CONDA

### For Change Control Board: Issue Review

- [X] Is it tagged with a type: defect or task?
- [X] Is it tagged with a priority: critical, normal or minor?
- [X] If it will impact requirements or requirements tests, is it tagged with requirements?
- [X] If it is a defect, can it cause wrong results for users? If so an email needs to be sent to the users.
- [X] Is a rationale provided? (Such as explaining why the improvement is needed or why current code is wrong.)

### For Change Control Board: Issue Closure

- [x] If the issue is a defect, is the defect fixed?
- [x] If the issue is a defect, is the defect tested for in the regression test system?  (If not explain why not.)
- [x] If the issue can impact users, has an email to the users group been written (the email should specify if the defect impacts stable or master)?
- [x] If the issue is a defect, does it impact the latest release branch? If yes, is there any issue tagged with release (create if needed)?
- [x] If the issue is being closed without a pull request, has an explanation of why it is being closed been provided?

### PR DESCRIPTION
**Pull request by wangcj05**
--------
Pull Request Description
--------
##### What issue does this change request address? (Use "#" before the issue to link it, i.e., #42.)
close #1711 

##### What are the significant changes in functionality due to this change request?

Fixes mistake in tests documentation that caused building pdf to fail.

Passes --verbose to recursive make for tests.

Removes warning when creating version.tex

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [x] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [x] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [x] 8. If an analytic test is changed/added is the the analytic documentation updated/added?
- [x] 9. If any test used as a basis for documentation examples (currently found in `raven/tests/framework/user_guide` and `raven/docs/workshop`) have been changed, the associated documentation must be reviewed and assured the text matches the example.

### CODE REVIEW

**Comment by wangcj05** [comment]
@joshua-cogliati-inl Can you take a look at this PR?

**Comment by wangcj05** [comment]
> I looked at the code and I have no objections, but I don't think any of these changes could solve any problems.
> (I have also been trying to debug things at #1713 )

I agree, thanks for the review. The changes in make_docs will make SAWTOOTH pass the docs check, and the other one is just to fix the typo in tests. @joshua-cogliati-inl

**Comment by joshua-cogliati-inl** [comment]
I found the mistake that was causing it to fail (and it was my fault).  I had left a ^ in test documentation.

I added the commit to fix it to this branch.

I also added a fix that allows --verbose to work properly for the tests directory.

File: dependencies.xml [hunk 1]
```diff
@@ -53,10 +53,7 @@ Note all install methods after "main" take
     <pyside2/>
     <nomkl os='linux' skip_check='True'/>
     <numexpr os='linux'/>
-    <ray os="mac,linux" source="pip">1.3</ray>
-    <!-- ray is not compatible with aioredis 2.0
-      https://github.com/ray-project/ray/issues/17475 -->
-    <aioredis os="mac,linux" source="pip">1.3</aioredis>
+    <ray os="mac,linux" source="pip"/>
```
Comment (joshua-cogliati-inl) [inline dependencies.xml:56]: Do we want to switch to an unversioned ray?

Comment (joshua-cogliati-inl) [reply dependencies.xml:56]: 1.8 sounds fine to me.

Comment (wangcj05) [reply dependencies.xml]: FYI, on our Mac test machine, ray 1.8 is not available. On my local mac machine, ray 1.8 can be found and installed. In this case, I will fix the ray version to 1.5. @joshua-cogliati-inl 
```
... Activating environment ...
... Installing libraries from PIP-ONLY ...
...pip-only command: pip install ray==1.8.* xmlschema
ERROR: Could not find a version that satisfies the requirement ray==1.8.* (from versions: 0.6.0, 0.6.1, 0.6.2, 0.6.3, 0.6.4, 0.6.5, 0.6.6, 0.7.0, 0.7.1, 0.7.2, 0.7.3, 0.7.4, 0.7.5, 0.7.6, 0.7.7, 0.8.0, 0.8.1, 0.8.2, 0.8.3, 0.8.4, 0.8.5, 0.8.6, 0.8.7, 1.0.0rc0, 1.0.0rc1, 1.0.0rc2, 1.0.0, 1.0.1, 1.0.1.post1, 1.1.0, 1.2.0, 1.3.0, 1.4.0rc1, 1.4.0rc2, 1.4.0, 1.4.1, 1.5.0, 1.5.1, 1.5.2)
ERROR: No matching distribution found for ray==1.8.*

ERROR: exiting with code 1
```

**Comment by wangcj05** [comment]
@joshua-cogliati-inl tests are ok now. I will ask @mandd to merge it.

**Review by mandd — APPROVED** [review]

**Comment by mandd** [comment]
Review board is green, sawtooth machine is now down due to maintenance activities.

**Pull request merged**